.github/workflows/integration-tests-amd.yml

```diff
@@ -104,16 +104,16 @@ jobs:
           pytest --capture=tee-sys -rfs third_party/amd/python/test/test_extract_slice_concat_op.py
 
           # Run gluon test for IR generation
-          TRITON_ALWAYS_COMPILE=1 pytest --capture=tee-sys -rfs python/test/gluon/test_frontend.py
+          pytest --capture=tee-sys -rfs python/test/gluon/
 
-          TRITON_ALWAYS_COMPILE=1 pytest --capture=tee-sys -rfs third_party/amd/python/test/test_scalarize_packed_fops.py
+          pytest --capture=tee-sys -rfs third_party/amd/python/test/test_scalarize_packed_fops.py
```

Comment:
We want to keep `TRITON_ALWAYS_COMPILE=1` for existing tests actually--there are reasons we need them there.